gensim/models/wrappers/fasttext.py

```diff
@@ -256,7 +257,14 @@ def load_binary_data(self, model_binary_file):
             self.load_vectors(f)
 
     def load_model_params(self, file_handle):
-        (dim, ws, epoch, minCount, neg, _, loss, model, bucket, minn, maxn, _, t) = self.struct_unpack(file_handle, '@12i1d')
+        magic, v= self.struct_unpack(file_handle, '@2i')
```

Comment:
Code style - keep 1 space before and after  `=`
Also, version would be preferable to `v`.